src/main/scala/tech/sourced/gemini/Gemini.scala

```diff
@@ -129,6 +129,25 @@ class Gemini(session: SparkSession, log: Slf4jLogger, keyspace: String = Gemini.
     ReportResult(duplicates, similarities)
   }
 
+  def isDBEmpty(session: Session, mode: String): Boolean = {
+    var row = session.execute(s"select count(*) from $keyspace.${tables.docFreq} where id='$mode'").one()
```

Comment:
is `select count(*)` performance ok to run this before every command? Would it improve if it was `select count(*) ... limit 1`?

Reply:
currently, it's okay because this table can contain only 2 rows max.
but it's a good point and better to update in case we change it. 👍

Reply:
ok, I missed that about the table :D

Reply:
applied